### ISSUE TITLE
how to split the dataset into training and validation sets

### ISSUE THREAD

**Issue by ycity-water** [XLearning-SCU/2024-NeurIPS-AverNet#1]
"Your work is great, and I am currently reproducing your code for learning purposes. Now, I have encountered a problem: you provided the dataset used for training, but you didn't explain how the training and validation sets are split. Could you clarify how this is done?"

**Comment by Pandint** [comment]
Hi! Thank you for your interest in our work!
In our implementation, we do not have a separate validation dataset. Instead, the validation set is created by randomly selecting a few video clips from the test set.

**Comment by ycity16** [comment]
Thank you for your reply. Regarding what you mentioned about randomly selecting video clips to create the dataset, I have another question. In your train_AverNet_DAVIS.yml file, there is a path for the validation dataset. Do I need to put any data in this path? When you mentioned randomly selecting data from the test set, do I need to manually select the data and place it in the validation path, or will it automatically select the data without me needing to add anything to the validation path?

**Comment by Pandint** [comment]
Yes, you need to manually select the data and place it in the path. 
For example, you can select 'man-bike', 'people-sunset', and 'salsa' in DAVIS-test as a validation dataset and put the corresponding input and ground-truth folders under the lq and gt paths, respectively.

**Comment by ycity16** [comment]
I understand now. Thank you very much for your response.
